docs/how-to/code/install-slice/task.yaml

```diff
@@ -46,7 +45,7 @@ execute: |
   sudo /snap/rockcraft/current/bin/skopeo --insecure-policy copy oci-archive:custom-openssl-rock_0.0.1_amd64.rock docker-daemon:chisel-openssl:latest
   # [docs:skopeo-copy-end]
 
-  rm -fr chisel/ chisel-releases/ my-custom-openssl-fs/ custom-openssl-rock_0.0.1_amd64.rock
+  rm -fr chisel-releases/ my-custom-openssl-fs/ custom-openssl-rock_0.0.1_amd64.rock
```

Comment:
I guess this `rm -fr $HOME/chisel-releases/` bit should be in the `restore` of both tests

Reply:
done in https://github.com/canonical/rockcraft/pull/456/commits/926ff933a1dbff61287c95620612c7f84da97cbc